modules/meeting/config/locales/en.yml

```diff
@@ -394,6 +394,14 @@ en:
     placeholder_title: "New section"
     empty_text: "Drag items here or create a new one"
 
+  meeting_participant:
+    participation_status:
+      needs_action: "No response"
+      accepted: "Accepted"
+      declined: "Declined"
+      tentative: "Maybe"
+      unknown: "Unknown"
```

Comment:
Nitpick: for the sake of discoverability, I would nest these enum translations under the model `activerecord.attributes.meeting_participant.participation_status`

Ideally we would have a convention for handling enum localisation. I'd like to see a `human_enum_name` method - I recall using https://github.com/jkostolansky/human_enum_name in the past, but it'd also be pretty straightforward for us to roll our own.